README.md

```diff
@@ -214,7 +214,15 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 
   It is also possible to create the K8s secrets (a commented out example is provided in the [kustomization.yaml](config/samples/microservices/kustomization.yaml)). To enable the secrets you'd need to uncomment the relevant secret generator entry and patch.
   
+- Working with PeerPods running in PeerPod Secure Comms 
 
+  These additional steps are needed to enable cloud-api-adaptor to connect as a KBS Client to the KBS.
```

Comment:
Wondering if this is specific for CAA to connect to KBS or any entity wanting to connect to KBS using kbs_client ?

Reply:
The steps create a secret that you can use by any `KBS Client`. 
It creates it in `confidential-containers-system` namespace rather than in `kbs-operator-system` namespace. 

kbs_client today does not read secrets from kubectl but instead expects to get the secret as a file 
Therefore, these steps right now are helpful to any code seeking to access KBS by approaching the KBS APIs and have access to secrets in the `confidential-containers-system` namespace